tests/integrationtests/tst_perfparser.cpp

```diff
@@ -242,10 +243,9 @@ private slots:
         QTest::addRow("PERF v1") << QFINDTESTDATA("file_content/perf.data.true.v1") << QStringLiteral("V1 perf data")
                                  << 1000;
 
-        // TODO: check why we need this long waittime
-        QTest::addRow("PERF v2") << QFINDTESTDATA("file_content/perf.data.true.v2") << QString() << 9000;
+        QTest::addRow("PERF v2") << QFINDTESTDATA("file_content/perf.data.true.v2") << QString() << 1000;
```

Comment:
ideally we just unify the test to always give it, say, 2s waittime and get rid of the custom wait time per data tag. it's just adding noise for no reason imo?

Reply:
We may need to adjust that if/when we add a "big" test case, so far we don't have it so I'm going to just use the magic 2000 in both places.